extensions/tmtv/src/commandsModule.js

```diff
@@ -60,9 +60,8 @@ const commandsModule = ({
       // that are in the viewer layout (but then we have the problem of the attenuation
       // corrected PT vs the non-attenuation correct PT)
       const matches = HangingProtocolService.getDisplaySetsMatchDetails();
-
       const matchedSeriesInstanceUIDs = Array.from(matches.values()).map(
-        ({ SeriesInstanceUID }) => SeriesInstanceUID
+        match => (match?.SeriesInstanceUID || '')
```

Comment:
Just fix an exception if the match comes up empty - this is a permitted situation, although it probably should cause a different HP to be applied, but there isn't a good way to do that here.